metagraph/src/graph/alignment/aligner_helper.hpp

```diff
@@ -473,8 +480,8 @@ class QueryAlignment {
     // vector may be incorrect, so this corrects them
     void fix_pointers(const std::string &query, const std::string &query_rc);
 
-    const std::string query_;
-    const std::string query_rc_;
+    std::string query_;
+    std::string query_rc_;
     std::vector<value_type> alignments_;
```

Comment:
Introducing this `value_type` makes it only harder to track what is stored. Remove it.

Reply:
I don't remember for sure, but I think `value_type` has to be defined for structured binding to work.

Reply:
If you added it to make some other feature like structured binding, you don't have to use it. It just makes more tangled. What is the point?

Reply:
I mean, you can introduce it, if it's necessary, but don't use it. Write the original type in this line.

Reply:
I tried it out now and it looks like it's not needed, having a `const_iterator` is good enough